DISCLAIMER-WIP

```diff
@@ -11,6 +11,7 @@ a list of known issues the project is currently aware of (note that this list, b
 is likely to be incomplete): 
 
  * Releases may have incomplete licensing conditions.
+ * Some ASF files don't have the correct headers.
```

Comment:
```suggestion
 * Some ASF files still have BSD headers.  Most of the top contributors have signed an ICLA and we are working on updating the headers and getting approvals from other copyright holders.
```
@btashton I pretty much copied what you wrote.
Please if you get anything to add/remove use the suggestions and we squash the commits when merging.

Reply:
Looks good to me. I think that makes the state of things clear for people outside the project which I think was Justin's request.